.circleci/config.yml

```diff
@@ -121,10 +121,10 @@ jobs:
       - install_tf
       - set_env_vars
       - run:
-          name: "Set module source to latest published release"
+          name: "Set module source to latest domino release"
           working_directory: tests
           command: |
-            latest_tag="$(git tag | tail -1)"
+            latest_tag="v2.1.1"
```

Comment:
@fraenkel Is this temporary? Why move away from a tag lookup?

Reply:
because it highlighted an issue with upgrade testing.
going from n-1 -> n while interesting isn't really useful.
We should be testing the big jump from Domino X-1 to what will be Domino X.

Reply:
I don't see how pinning to the upcoming version number as a static string fixes this, and it also introduces a manual update task from here forward if we don't want to depend on this one release forever.

Reply:
I realize that, but the reverse is actually worse since no one is doing that upgrade.
For example, just this PR, would only test that we can switch the name from platform -> compute.
That in and of itself isn't useful. Over time, the single updates are not interesting because unless someone does that Domino upgrade, we don't know when we are actually breaking things.
Whether we actually update the version is somewhat up for debate. The bigger the jump we guarantee the better off we are.

Reply:
I am okay with moving forward with this and am logging a more general review of the tests in the backlog.